README.md

```diff
@@ -62,6 +62,20 @@ This mode allows to visualize images received on the topics specified under `ima
 The topic manager can add and remove topics int the termviz config. When confirmed the config will be stored and termviz must be restarted.
 Only supported topics are displayed, topics can only be in the active or in the available list.
 
+### TF Tree View
+
+Similar to rqt_tf_tree, it displays the TF tree and allows users to interactively echo any two arbitrary frames.
+
+#### Prerequisites
+
+Ensure the tf2_frames service is available by running the [tf_service server](https://github.com/magazino/tf_service) with the --frames_service argument:
+
+```sh
+rosrun tf_service server --frames_service
+```
+**Note**: tf service server is required since [rustros_tf](https://github.com/arjo129/rustros_tf) doesn't advertise by default tf2_frames service like C++/Python versions do.
```

Comment:
Gramma makes it hard to understand here "does not advertise the tf2_frames service by default."